### ISSUE TITLE
Search tab titles as well as session titles

### ISSUE THREAD

**Issue by Nezteb** [ReDEnergy/SessionSync#82]
Currently, SessionSync's search bar only searches the user-given (or auto-generated) titles of sessions. 

![Screen Shot 2019-12-07 at 11 21 02 PM](https://user-images.githubusercontent.com/3588798/70384699-923c8180-1948-11ea-96dc-e04e6537669d.png)

Similar extensions (SessionBuddy for Chrome) also search all tab titles contained in each session, which is very useful when trying to find specific tabs.

I know this could be a privacy concern (reading tab titles), but it could be a user-configurable choice.

**Comment by ReDEnergy** [comment]
Yes, I know... this is a feature that should have been from the start. It's just not that easy to implement and I was not able to find the time to implement it. I can only promise that's something I definitely want to add and it will be integrated at some point.

**Comment by Nezteb** [comment]
It's not a huge issue; I can still use Firefox's built-in bookmark search to find things I've saved with SessionSync. I just thought I'd mention/document the request here.

This extension is amazing and I love it! If I find time in the next few months, I might make a PR to attempt to implement this feature myself!
